ambry-api/src/main/java/com/github/ambry/account/Container.java

```diff
@@ -61,6 +61,7 @@
   static final String CONTAINER_NAME_KEY = "containerName";
   static final String CONTAINER_ID_KEY = "containerId";
   static final String STATUS_KEY = "status";
+  static final String CONTAINER_LAST_UPDATE_TIME_KEY = "lastUpdateTime";
```

Comment:
The PR looks good after addressing people's comments.

I have one concern:  `lastUpdateTime` sounds like a general update time. Any update can refer to it. 
But in this PR, it is used as delete triggered time.

Reply:
I named the field in a general way intentionally cause for container deletion the lastUpdateTime should be when user mark container as DELETE_IN_PROGRESS(since after user mark the status, ACLs would be removed and they can't access this container). But for future we may use this field for different purpose. What do you think? If naming style should be more specific, I can change to deleteTriggerTime.

Reply:
If it's for general use cases, you might need to update this value for all container related update. Not only ACL changes, but all other container metadata changes. I prefer a narrow name:  `deleteTriggerTime `. @justinlin-linkedin @cgtz  any opinion?

Reply:
I am fine with deleteTriggerTime. From my past experience, we actually don't update  container that much, except for adding download content type in the whitelist and delete container. Of course we can update the container in anyway allowed but we just don't. And we don't even have a creation time, which indicates we just don't care about update time at all. So a delete time should be enough for a container.
That being said, if we start to care more about when the container is created, update and deleted, then, we might have to at least have a creation time and update time.

Reply:
Thank you all for the great explanation. Updated to deleteTriggerTime.